standup.py

```diff
@@ -98,6 +98,29 @@ def standup_teams_remove(self, msg, name):
             teams.remove(team)
         return 'Team removed.'
 
+    @arg_botcmd('email', type=str)
+    @arg_botcmd('name', type=str, default=None)
+    def standup_team_configure(self, msg, email, name):
```

Comment:
shouldn't it be name then email?

Reply:
If the order is switched, how would you specify just the e-mail and deduce the team name from the room?

Reply:
hmm ok good point